ashpy/keras/losses.py

```diff
@@ -263,6 +263,5 @@ def reduction(self, value: tf.keras.losses.Reduction) -> None:
 
     def call(self, d_real: tf.Tensor, d_fake: tf.Tensor) -> tf.Tensor:
         """Computes the hinge loss"""
```

Comment:
@EmanueleGhelfi can you update the documentation of the `__init__` method of this class, explaining **why** the hinge-loss is computed in this way for the generator?

Reply:
Or better, the class documentation and not the `__init__` doc